packages/react/src/VideoPlayer/VideoPlayer.module.css

```diff
@@ -0,0 +1,398 @@
+/* ---------------------------------------------------------- */
+/* VideoPlayer Contents                                       */
+/* ---------------------------------------------------------- */
+/* Table of contents:                                         */
+/* 1. Container                                               */
+/* 2. Player                                                  */
+/* 2. Center Play Button                                      */
+/* 4. Title                                                   */
+/* 5. Controls                                                */
+/* 6. Range Element                                           */
+/* 7. Tooltip                                                 */
+/* 8. Captions                                                */
+/* ---------------------------------------------------------- */
+
+/* ---------------------------------------------------------- */
+/* 1. Container                                               */
+/* ---------------------------------------------------------- */
+
+.VideoPlayer__container {
+  width: 100%;
+  position: relative;
+  overflow: hidden;
+}
+
+/* ---------------------------------------------------------- */
+/* 2. Player                                                  */
+/* ---------------------------------------------------------- */
+
+.VideoPlayer {
+  width: 100%;
+  border-radius: var(--brand-borderRadius-medium);
+  display: block;
+}
+
+/* ---------------------------------------------------------- */
+/* 3. Center Play Button                                      */
+/* ---------------------------------------------------------- */
+
+.VideoPlayer__playButton {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0);
```

Comment:
Should this just be `transparent`? Or not set at all?

Reply:
This has been updated in this SHA https://github.com/primer/brand/pull/373/commits/ae69cf85dd5659e66feca9b7586ee79ae7c33dee